internal/cli/cluster/helper.go

```diff
@@ -302,19 +301,35 @@ func BuildStorageClass(storages []StorageInfo) string {
 	return util.CheckEmpty(strings.Join(scs, "\n"))
 }
 
-// GetLatestVersion gets the latest cluster versions that referencing the cluster definition
-func GetLatestVersion(dynamic dynamic.Interface, clusterDef string) (string, error) {
+// GetDefaultVersion gets the default cluster version that referencing the cluster definition.
+// If only one version is found, it will be returned directly, otherwise the version with
+// constant.DefaultClusterVersionAnnotationKey label will be returned.
+func GetDefaultVersion(dynamic dynamic.Interface, clusterDef string) (string, error) {
 	versionList, err := GetVersionByClusterDef(dynamic, clusterDef)
 	if err != nil {
 		return "", err
 	}
 
-	// find the latest version to use
-	version := findLatestVersion(versionList)
-	if version == nil {
-		return "", fmt.Errorf("failed to find the latest cluster version referencing current cluster definition %s", clusterDef)
+	if len(versionList.Items) == 1 {
+		return versionList.Items[0].Name, nil
 	}
-	return version.Name, nil
+
+	found := false
```

Comment:
is var `found` redundant here ?  we can tell the truth from `defaultVersion`.  (found = (defaultVersion != ""))

Reply:
Yes, fixed.